latex/paper.tex

```diff
@@ -65,20 +66,20 @@
 
 \section{Introduction}
 % Context
-The recent rise in data sharing and improved data collection strategies
-have brought neuroimaging to the Big Data era~\cite{ALFAROALMAGRO:18,
-van2014human}. Existing neuroimaging workflow engines, such as
-Nipype~\cite{Nipype:11}, are well suited for processing the standard
-compute-intensive neuroimaging pipelines, but lack incorporated Big Data
-strategies (i.e. in-memory computing, data locality, and lazy-evaluation) to
-improve performance of the increasingly prevalent data-intensive pipelines.
-As was noted in~\cite{hayot2019performance}, in-memory computing, coupled
-with data locality, can bring significant performance improvements to data
-intensive neuroimaging pipelines. We extend this work by studying the
+The recent rise in data sharing and improved data collection strategies have brought
+neuroimaging to the Big Data era~\cite{ALFAROALMAGRO:18, van2014human}. Existing
+neuroimaging workflow engines, such as Nipype~\cite{Nipype:11}, are well suited for
+processing the standard compute-intensive neuroimaging pipelines, but lack
+incorporated Big Data strategies (i.e. in-memory computing, data locality, and
+lazy-evaluation) to improve performance of the increasingly prevalent data-intensive
+pipelines. As was noted in~\cite{hayot2019performance}, in-memory computing, coupled
+with data locality, can bring significant performance improvements to data intensive
+neuroimaging pipelines. We are interested in the impact on performance when
+implementing these Big Data strategies in different engines. Our work studies the
```

Comment:
I would be more explicit: replace "We are interested ... " with:
"In this work, we investigate how performance benchmarks generalize across Big Data engines. In particular, we study the differences between Dask and Apache Spark, ...".